.github/workflows/clingo.yml

```diff
@@ -50,7 +55,7 @@ jobs:
     runs-on: ["self-hosted", "macOS", "ARM64"]
     strategy:
       matrix:
-        python-version: ["3.6.15", "3.7.14", "3.8.14", "3.9.14", "3.10.7", "3.11.0"]
+        python-version: ["3.6.15", "3.7.14", "3.8.14", "3.9.14", "3.10.7", "3.11.5", "3.12.0"]
```

Comment:
Why is this pinned to patch version number?

Reply:
Because it's on a self-hosted runner and python installations are managed with `pyenv`